addons/web_editor/tests/test_views.py

```diff
@@ -62,3 +62,15 @@ def test_oe_structure_as_inherited_view(self):
             '<p>Hello World!</p>',
             base.inherit_children_ids.read_combined(['arch'])['arch'],
         )
+
+    def test_find_available_name(self):
+        View = self.env['ir.ui.view']
+        used_names = ['Unrelated name']
+        initial_name = "Test name"
+        name = View._find_available_name(initial_name, used_names)
+        self.assertEqual(initial_name, name)
+        used_names.append(name)
+        for count in range(2, 10):
+            name = View._find_available_name(initial_name, used_names)
+            self.assertEqual('%s (%s)' % (initial_name, count), name)
+            used_names.append(name)
```

Comment:
Not a specialist in python tests... seems really overkill to me to test from 2 to 10 but why not? Actually this is a question for myself, @JKE-be ?